otter/integration/config.sh-template

```diff
@@ -51,11 +51,11 @@ export AS_NOVA_SC_KEY=cloudServersOpenStack
 export AS_CLB_SC_KEY=cloudLoadBalancers
 
 # Uncomment this if you are running tests against Mimic.
```

Comment:
Maybe "comment this out if you are not running tests..." - similarly with the next comment?